dualscreeninfo/src/twopaneview/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+export { RNHinge as Hinge } from './RNHinge'
+export { RNTwoPaneView as TwoPaneView } from './TwoPaneView'
```

Comment:
We want to publish 2 separate packages out of this monrepo:
1) A lower-level package with just the dualscreeninfo native module.  This will be used by customers who aren't using TwoPaneView but want to enlighten their apps.
2) A package for TwoPaneView.

Reply:
Not sure what is the reason to split it

Reply:
Also where then to put Hinge component?

Reply:
It could be published with TwoPaneView, but I personally feel we don't need a Hinge class at all.  We can talk more about this as Homero digs into the TwoPaneView implementation.

---
In reply to: [382734643](https://github.com/microsoft/react-native-dualscreen/pull/2#discussion_r382734643) [](ancestors = 382734643)